wordpress-mobile/WordPress-iOS
# Issue/6219 allow dismissis of post type filter

Fixes #6219

To test:
 - Go to Blog->Post
 - Select the filter post type on the top of the view
 - Try to tap outside of the filter list to dismiss the picker
 - Test this on the iPhone+ devices, but also on the iPads and regular iPhone to see if all works ok.


Needs review: @aerych
